platform/mellanox/mlnx-platform-api/sonic_platform/chassis.py

```diff
@@ -28,6 +30,35 @@
 
 GET_HWSKU_CMD = "sonic-cfggen -d -v DEVICE_METADATA.localhost.hwsku"
 
+PLATFORM_ROOT = '/usr/share/sonic/device'
+EEPROM_CACHE_ROOT = '/var/cache/sonic/decode-syseeprom'
+EEPROM_CACHE_FILE = 'syseeprom_cache'
+
+HWMGMT_SYSTEM_ROOT = '/var/run/hw-management/system/'
+
+REBOOT_CAUSE_ROOT = HWMGMT_SYSTEM_ROOT
+
+REBOOT_CAUSE_POWER_LOSS_FILE = 'reset_main_pwr_fail'
+REBOOT_CAUSE_THERMAL_OVERLOAD_ASIC_FILE = 'reset_asic_thermal'
+REBOOT_CAUSE_WATCHDOG_FILE = 'reset_hotswap_or_wd'
+REBOOT_CAUSE_MLNX_FIRMWARE_RESET = 'reset_fw_reset'
```

Comment:
can we find a better name for reset_fw_reset? sounds confusing

Reply:
These are not the strings that represent reboot reasons to users but the names of files that are in the /var/run/hw-management/ and for us to retrieve the reboot reason, so I don't prefer to change their names.